### DISCUSSION TITLE
Questions on CRS Choices

### DISCUSSION THREAD

**Discussion by rohannallamadge** [OSGeo/grass#4832] [Q&A]
Hello,

I am excited to contribute to GRASS GIS and have been working with hydrological modeling tools like `r.sim.water` and `r.sim.sediment`. I would appreciate some guidance on the following questions:

Common CRS Choices for Hydrological Modeling

I'm trying to determine the best Coordinate Reference System (CRS) for sediment modeling workflows, especially when using datasets like the North Carolina full dataset. Are there specific CRS options that are typically recommended for` r.sim.sediment`? Any advice on CRS choices would be greatly appreciated.
`r.sim.sediment` Workflow

I am working on using the water depth results from `r.sim.water` to simulate erosion and deposition using` r.sim.sediment`. Are there any key considerations, best practices, or resources that can help me set up this workflow more effectively?
I am very enthusiastic about contributing to GRASS GIS and improving my skills with these tools. If there are any additional examples or references that you could point me toward, I would be grateful.

Also, is it okay if I reach out in the future with more questions as I continue working on these projects?

Thank you for your time and support!

Best regards,
Rohan

**Comment by petrasovaa** [comment]
CRS should be projected with units meters (or feet), not degrees. I think you need to be more specific in your other question (as I replied in email earlier).
